content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -79,6 +79,30 @@ spec:
 EOF
 {{< /text >}}
 
+{{< tip >}}
+If you installed Istio using Helm, the selector for the `Gateway` is `istio=ingress`:
```

Comment:
This is not accurate, and one of the reasons we changed the label in the install doc :-) 

Its just a standard K8s label selector. There is nothing special about it, at all. Users need to know this, and not treat it as a magic value.

The label on the ingress is user defined. Just because our docs examples happens to set it to `istio=ingress` is not super important. A parallel would be a K8s doc deploying a Pod with app=foo, then making every Service doc say "set `selector: app=foo`" - it works for the example, but doesn't teach users what they need to know.

Reply:
Gateway: arbitrary label selector.
Helm chart: users can set whatever labels they want.

I would be fine with saying "The selector matches pod labels; if you followed the standard helm docs <LINK> this would be `istio=ingress`". Just think we need to emphasize its not a magic value - just a label selector.

Reply:
Ah ok yeah I see what you mean. Yeah my initial phrasing was poorly worded.